com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/protocol/DefinitionResult.java

```diff
@@ -5,17 +5,19 @@
 
 package com.microsoft.java.lsif.core.internal.protocol;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class DefinitionResult extends Vertex {
 
-	// TODO: Support bag result.
-	private String result;
+	private List<String> result;
 
 	public DefinitionResult(String id, String result) {

```

Comment:
No need to change param? String[]?

Reply:
That's a good idea. But since symbols in Java will map to one definition. I think the API can leave it as it is so far.